www/js/main-layout.controller.js

```diff
@@ -19,7 +19,7 @@ angular
             vm.accounts = accountService.getAll();
             currentAccount = accountService.getDefault();
             if (currentAccount && _.keys(currentAccount).length) {
-                const landingCompany = localStorage.getItem('landingCompany');
+                const landingCompany = currentAccount.landing_company_name || localStorage.getItem('landingCompany');
```

Comment:
@nazaninreihani 
Is there any cases that `currentAccount.landing_company_name` doesn't have value or its value is different from `localstorage.landingCompany`?

as I check every time both of them are as same. am I right?

Reply:
Yes they are always the same, I will use only one of them